### ISSUE TITLE
OTP problem

### ISSUE THREAD

**Issue by centauruspool** [andreaswilli/react-verification-input#6]
When the user receives the sms code, you can see it on top of your keyboard inside you mobile app. If you press the code, it gets sent to the input, but only in that white box that you see when you have the debug set as true.

**Comment by andreaswilli** [comment]
Thanks for the report. Is this happening on an iPhone or on Android? Also, do you know how this can be tested (e.g. iPhone simulator etc.)?

**Comment by centauruspool** [comment]
I only tested it on an iPhone with a sim card (so I could receive the SMS code).

Another hint would be that when you press one number on the keyboard, the rest of the numbers from the sms code are shown in the input as well.

**Comment by andreaswilli** [comment]
What's your exact setup? I just tried with this option and it seems to work fine:
```jsx
<VerificationInput inputField={{ autocomplete: "one-time-code" }} />
```
![otp](https://user-images.githubusercontent.com/17298270/82094133-bfae0800-96fc-11ea-8e4b-1664e43c5ff4.gif)

**Comment by centauruspool** [comment]
Hmm, weird. 

`<VerificationInput
          character={{ classNameSelected: 'character--selected' }}
          input={{ onChange: value => this.setState({ code: value }) }}
          inputField={{
            autoComplete: 'one-time-code',
            type: 'tel',
          }}
          placeholder="."
          removeDefaultStyles
          validChars={'0-9'}
        />`

**Comment by centauruspool** [comment]
The issue is that the 'button' on the keyboard that has the OTP attached to it, does NOT have a `keycode`. So when you press it, the `onKeyUp` event, doesn't get triggered.

**Comment by andreaswilli** [comment]
This should be resolved in `v2.0.0` as it does no longer rely on the `keyUp` event to detect changes.
